scripts/strip-frameworks.sh

```diff
@@ -60,3 +60,11 @@ for file in $(find . -type f -perm +111); do
     fi
   fi
 done
+
+if [ "$ACTION" = "install" ]; then
+  echo "Copy .bcsymbolmap files to .xcarchive"
+  find . -name '*.bcsymbolmap' -type f -exec mv {} "${CONFIGURATION_BUILD_DIR}" \;
```

Comment:
@kishikawakatsumi I've generalized this so `strip-frameworks.sh` works for any framework, like the rest of this script. Could you please test this?